examples/pytorch/rgcn/experimental/README.md

```diff
@@ -75,15 +111,41 @@ python ~/dgl/tools/copy_files.py --ip_config ip_config.txt \
 
 Users need to copy the training script to the workspace directory on remote machines as well.
 
+
+</p>
+</details>
+
+
+
+### Step 1: set IP configuration file.
+
+User need to set their own IP configuration file before training. For example, if we have four machines in current cluster, the IP configuration could like this:
+
+```bash
+172.31.0.1
+172.31.0.2
+172.31.0.3
+172.31.0.4
+```
+
+Users need to make sure that the master node (node-0) has right permission to ssh to all the other nodes.
+
+### Step 2: partition the graph.
+
+The example provides a script to partition some builtin graphs such as ogbn-mag graph.
+If we want to train RGCN on 4 machines, we need to partition the graph into 4 parts.
+
+In this example, we partition the ogbn-mag graph into 4 parts with Metis. The partitions are balanced with respect to
+the number of nodes, the number of edges and the number of labelled nodes.
+```bash
+python3 partition_graph.py --dataset ogbn-mag --num_parts 4 --balance_train --balance_edges
+```
+
 ### Step 3: Launch distributed jobs
 
 DGL provides a script to launch the training job in the cluster. `part_config` and `ip_config`
 specify relative paths to the path of the workspace.
 
-The command below launches one training process on each machine and each training process has 4 sampling processes.
-**Note**: There is a known bug in Python 3.8. The training process hangs when running multiple sampling processes for each training process.
-Please set the number of sampling processes to 0 if you are using Python 3.8.
```

Comment:
we have to notify users of how to set up the sampling processes correctly. we'll fix the bug in the next release.